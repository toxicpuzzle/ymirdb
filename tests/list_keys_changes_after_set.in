LIST KEYS
SEt a 1 2 3
SEt b 2 23 4
LIST KEYS
DEL a
LIST KEYS
bye