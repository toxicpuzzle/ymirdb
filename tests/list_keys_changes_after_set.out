> no keys

> ok

> ok

> b
a

> ok

> b

> bye
